### PR TITLE
add snapshot export documentation, rewrite basic usage

### PR DESCRIPTION
**Pull request by LesnyRumcajs**
## Summary of changes

Changes introduced in this pull request:

- added snapshot export documentation as per @jennijuju suggestion,
- rewritten basic usage page to reflect the current state-of-the-art

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: documentation/src/basic_usage.md [hunk 1]
```diff
@@ -1,68 +1,87 @@
...
+```shell
+sudo apt install build-essential clang cmake
 ```
 
-### Commands
+### Optional runtime dependencies
 
-```bash
-make release
+[aria2](https://aria2.github.io/) is an alternate backed for downloading the
```
Comment (creativcoder) [inline documentation/src/basic_usage.md:27]: s/backed/backend ?

**Review by creativcoder — APPROVED** [review]

**Review by lemmih — APPROVED** [review]
Do we still depend on `cmake`? I don't think I have it installed.

File: documentation/src/basic_usage.md (hunk 1 above)
Comment (LesnyRumcajs) [reply documentation/src/basic_usage.md]: Thanks!

**Comment by LesnyRumcajs** [comment]
> Do we still depend on `cmake`? I don't think I have it installed.

I'll have a look and if it is the case, I'll remove it from here and the Dockerfiles.

**Pull request merged**